lib/rules/no-unused-services.js

```diff
@@ -0,0 +1,203 @@
+'use strict';
+
+const types = require('../utils/types');
+const emberUtils = require('../utils/ember');
+const propertyGetterUtils = require('../utils/property-getter');
+
+function getErrorMessage(serviceName) {
+  return `The service \`${serviceName}\` is not referenced in this JS file, but it may still be referenced in the corresponding HBS file. Please remove this service if it is not being used within the HBS file.`;
```

Comment:
```suggestion
  return `The service \`${serviceName}\` is not referenced in this JS file, but it may still be referenced in the corresponding HBS file. Please remove this service injection if it is not being used within the HBS file.`;
```